extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ConnectionModel.java

```diff
@@ -165,6 +172,28 @@ public Integer getMaxConnectionsPerUser() {
     }
 
     /**
+     * Sets the connection weight.
+     *
+     * @param connectionWeight
+     *     The weight of the connection.  null is acceptable, and anything
+     *     less than one will prevent the connection from being used.
+     */
+    public void setConnectionWeight(Integer connectionWeight) {
+        this.connectionWeight = connectionWeight;
+    }
+
+    /**
+     * Returns the connection weight used in calculating the
+     * weighted algorithms.
+     *
+     * @return
+     *     The connection weight as an Integer.
```

Comment:
To avoid confusion, I suggest removing "as an Integer" here, as that wording implies that the connection weight is a different type under other circumstances. If the intent is to document the return type, that's already inherently documented by the function prototype.